src/components/AnalysisContent/index.js

```diff
@@ -98,26 +97,16 @@ function AnalysisContent({
         return indicator;
       });
     }
-
-    const hurumapCharts = Array.from(
-      document.querySelectorAll('div[data-chart-id]')
+    setHurumapCharts(
+      Array.from(document.querySelectorAll('div[data-chart-id]')).map(
```

Comment:
QQ: Shouldn't `data-chart-id` have been `data-hurumap-chart-id` or something? I'm just thinking when we have HURUmap and Flourish on the page and we need to _hydrate_ both of them.